ramls/customField.json

```diff
@@ -76,6 +76,10 @@
       "description": "Text field properties",
       "$ref": "textField.json"
     },
+    "displayInAccordion": {
+      "type": "string",
+      "description": "Display in accordion type names for user entity type"
```

Comment:
The field description may need to be revised now that it's not exclusively associated with user values.